client/me/help/help-contact/index.jsx

```diff
@@ -260,7 +263,13 @@ module.exports = React.createClass( {
 		}
 
 		if ( ! ( olark.isOlarkReady && sitesInitialized ) ) {
-			return <div className="help-contact__placeholder" />;
+			return (
```

Comment:
How about you use the actual contact form and just add the placeholder styles.
do something like:

```js
return (
    <div className="help-contact__placeholder">
        <HelpContactForm disabled={true} showSubjectField={ true } />
    </div>
);
```